src/components/robot/Client.ts

```diff
@@ -0,0 +1,175 @@
+import type Client from '../../Client'
```

Comment:
Let's remove the robot interface and client for now

Reply:
Done